web-ui/components/Editor.vue

```diff
@@ -57,10 +70,49 @@ export default {
       },
     };
   },
+  mounted: function () {
+    let windowWidth = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
+    this.percentInPx = windowWidth / 100;
+    window.addEventListener("mousemove", (e) => this.resizeEditor(e));
+    window.addEventListener("click", this.stopDrag);
+    window.addEventListener('resize', () => this.updateWindowDimensions());
+  },
+  beforeDestroy() {
+    window.removeEventListener("mousemove", (e) => this.resizeEditor(e));
+    window.removeEventListener("click", this.stopDrag);
+    window.removeEventListener('resize', () => this.updateWindowDimensions());
+  },
   methods: {
     handleTab(tab) {
-      this.tab = tab;
+      this.tab = tab; 
     },
+    updateWindowDimensions(e) {
+      let windowWidth = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
+      this.percentInPx = windowWidth / 100;
+    },
+    startDrag() {
+      this.isResizing = true;
+    },
+    stopDrag() {
+      if(this.isResizing)
+        document.documentElement.style.setProperty('--output-width', `${100 - this.mouseCurrentX/this.percentInPx}%`);
```

Comment:
We don't need to set the property in stopDrag, it has already been set when we were dragging and this causes a small jump in the position of the windows.